torchgeo/models/resnet.py

```diff
@@ -18,6 +18,92 @@
     _satlas_transforms,
 )
 
+# Landsat TM-toa 7 channels
+bands_7_channels_tm_toa = [
```

Comment:
I would name this something like `_landsat_tm_toa_bands`, similar for other variables